entities/schema/platformics.yaml

```diff
@@ -113,8 +113,8 @@ enums:
       level_superkingdom:
   BulkDownloadType:
     permissible_values:
-      concatenate:
-      zip:
+      consensus_genome:
+      consensus_genome_intermediate_output_files:
```

Comment:
These correspond to the options the user sees in the bulk download modal: 
![Screenshot 2024-03-20 at 2 11 15 PM](https://github.com/chanzuckerberg/czid-platformics/assets/53838890/0973969f-cc88-4454-a243-4e044e969657)


This is also consistent with what rails uses: https://github.com/chanzuckerberg/czid-web-private/blob/19ea376c82f88d713c08892ca2ed4548ea7d64e3/app/helpers/bulk_download_types_helper.rb#L24

The "Sample Metadata" and "Consensus Genome Overview" downloads don't go through this flow, so they're not included here.